src/lua/help_en_US.lua

```diff
@@ -1,6 +1,16 @@
---- help (en_US)
+--- help (en_US) 1.10
 
 return {
+    help = [[
```

Comment:
It is not used in 1.10 and just ignored. See https://travis-ci.org/tarantool/tarantool/jobs/506834213#L2504 . The reason is that c3afd01ab607e571fcfa986770d61aa9cb8cd668 (backslash-commands) was not pushed to 1.10, because the primary need in these commands was lua/sql switching. So the description of these commands below is not valid for 1.10.